Misc/NEWS.d/next/Library/2020-10-26-14-17-44.bpo-41818.5MgaEc.rst

```diff
@@ -0,0 +1 @@
+Updated tests for the pty library.
```

Comment:
The news entry should optimally refer to the tests that were changed (in this case `test_opentty` was updated and a new `test_master_read()` was added.

Reply:
Sure!